### PR TITLE
Plant release notes for release 0.3.0

### PR DESCRIPTION
**Pull request by lucafavatella**
TODO
* [ ] Release notes for 0.3.0 (this PR)
* [x] Update any links in https://github.com/aeternity/epoch/releases/tag/v0.2.0-good-peers
  * [x] Ensure RELEASE-NOTES.md pointed by https://github.com/aeternity/epoch/releases/tag/v0.2.0-good-peers does not point back to master. See PR #401
  * [x] Update link in https://github.com/aeternity/epoch/releases/tag/v0.2.0-good-peers to cfe5c12 (i.e. master with PR 401 merged)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,164 +1,42 @@
...
 
-When a node managed to mine a block but such block is already obsolete because the local view of the longest chain moved ahead, you shall see something like the following on that node:
+### Starting node
 
-    2017-10-06 14:55:38.833 [error] <0.667.0>@aec_miner:running:157 Header insertion failed: {previous_hash_is_not_top,{top_header,{header,1,<<187,180,105,28,188,229,33,158,42,170,159,167,5,88,27,71,239,65,60,137,90,246,23,39,9,221,215,191,241,248,189,71>>,<<191,174,187,89,108,27,140,42,105,221,167,30,148,41,174,167,139,170,47,89,148,212,116,169,233,137,155,145,130,127,55,35>>,<<92,56,191,177,109,146,202,185,102,235,250,36,241,141,84,92,100,4,227,220,239,5,184,47,28,115,207,92,232,216,81,30>>,553713663,1125596692,1507300094807,1,[21444,6586407,9734926,17062307,19214768,19457624,20388691,22871531,22915558,35146174,37582353,41094493,50512682,50948743,52048504,54359502,58783324,67452260,67905997,72675145,76864212,78278965,80449837,81328841,91880678,93765245,95533733,97720772,101588269,102818356,102936000,107595015,107815273,110989552,112066712,113045125,114172277,115303768,118573306,125832620,127431107,128332518]}}}.
+    cd /tmp/node1
+    bin/epoch start
```
Comment (dincho) [inline RELEASE-NOTES.md]: shouldn't it be ./bin/epoch start ?

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,164 +1,42 @@
...
 
-When a node managed to mine a block but such block is already obsolete because the local view of the longest chain moved ahead, you shall see something like the following on that node:
+### Starting node
 
-    2017-10-06 14:55:38.833 [error] <0.667.0>@aec_miner:running:157 Header insertion failed: {previous_hash_is_not_top,{top_header,{header,1,<<187,180,105,28,188,229,33,158,42,170,159,167,5,88,27,71,239,65,60,137,90,246,23,39,9,221,215,191,241,248,189,71>>,<<191,174,187,89,108,27,140,42,105,221,167,30,148,41,174,167,139,170,47,89,148,212,116,169,233,137,155,145,130,127,55,35>>,<<92,56,191,177,109,146,202,185,102,235,250,36,241,141,84,92,100,4,227,220,239,5,184,47,28,115,207,92,232,216,81,30>>,553713663,1125596692,1507300094807,1,[21444,6586407,9734926,17062307,19214768,19457624,20388691,22871531,22915558,35146174,37582353,41094493,50512682,50948743,52048504,54359502,58783324,67452260,67905997,72675145,76864212,78278965,80449837,81328841,91880678,93765245,95533733,97720772,101588269,102818356,102936000,107595015,107815273,110989552,112066712,113045125,114172277,115303768,118573306,125832620,127431107,128332518]}}}.
+    cd /tmp/node1
```
Comment (dincho) [inline RELEASE-NOTES.md]: It would be nice if we wrap all bash commands in code blocks.

**Comment by dincho** [comment]
> Change https://github.com/aeternity/epoch/blob/master/RELEASE-NOTES.md to https://github.com/aeternity/epoch/blob/1be1208c561cfdae856e40ce6bf86ff42b4549ec/RELEASE-NOTES.md

Using link based on the tag itself sounds better to me e.g. https://github.com/aeternity/epoch/blob/v0.2.0-good-peers/RELEASE-NOTES.md

**Comment by lucafavatella** [comment]
> Using link based on the tag itself sounds better to me e.g. https://github.com/aeternity/epoch/blob/v0.2.0-good-peers/RELEASE-NOTES.md

I need to check whether the doc were updated after the tag.

And actually it is more complex than that as that file refers to master. Let me prepare separate PR for not breaking notes of release 0.2.0.

**Comment by lucafavatella** [comment]
I see OSX package at https://github.com/aeternity/epoch/releases/download/v0.3.0-big-spenders/epoch-0.3.0-osx-10.12.6.tar.gz
I updated release notes with 5 -> 6 change.

**Pull request merged**